sdk/core/package-lock.json

```diff
@@ -10,8 +10,13 @@
       "license": "ISC",
       "dependencies": {
         "@affinidi/affinity-metrics-lib": "^0.0.25",
+        "@affinidi/common": "^2.0.0-beta.9",
```

Comment:
Please never use `npm install` inside project directories; only use `npm run bootstrap` from the root monorepo directory.